src/Symfony/Component/Yaml/Tests/InlineTest.php

```diff
@@ -401,4 +401,13 @@ public function getTestsForDump()
             array('{ foo: { bar: { 1: 2, baz: 3 } } }', array('foo' => array('bar' => array(1 => 2, 'baz' => 3)))),
         );
     }
+
+    /**
+     * @expectedException \Symfony\Component\Yaml\Exception\ParseException
+     * @expectedExceptionMessage Malformed inline YAML string: {this, is not, yaml}.
+     */
+    public function testStringOffsetCastError()
+    {
+        Inline::parse('{this, is not, yaml}');
```

Comment:
This is valid yaml, to not confuse people what about `{this, is, not, supported}`?